cmd/osm-bootstrap/crds/config_meshconfig.yaml

```diff
@@ -99,12 +99,10 @@ spec:
                         - TLSv1_3
                       default: TLSv1_2
                     tlsMaxProtocolVersion:
-                      description: The maximum TLS protocol version that the sidecar supports. Valid TLS protocol versions are TLS_AUTO, TLSv1_0, TLSv1_1, TLSv1_2 and TLSv1_3.
+                      description: The maximum TLS protocol version that the sidecar supports. Valid TLS protocol versions are TLS_AUTO, TLSv1_2 and TLSv1_3.
                       type: string
                       enum:
                         - TLS_AUTO
-                        - TLSv1_0
```

Comment:
We actually can't remove fields from the enum; it would be a breaking change. This will need to be a documentation change only